test/delayed_queue_test.rb

```diff
@@ -383,7 +377,7 @@
     Resque.enqueue_at(t, SomeIvarJob, 'baz')
 
     assert_equal(2, Resque.remove_delayed(SomeIvarJob, 'bar'))
-    assert_equal(1, Resque.delayed_queue_schedule_size)
+    assert_equal(2, Resque.delayed_queue_schedule_size)
```

Comment:
how did this pass before? can we `assert_equal(4, Resque.delayed_queue_schedule_size)` before L379?

Reply:
The reason it passed before is a change in what `delayed_queue_size` meant. Previously it would have been `1` since they were all enqueued at the same time.